documentation/restarting.markdown

```diff
@@ -7,15 +7,15 @@ canonical: "/puppetserver/latest/restarting.html"
 [logback.xml]: ./config_file_logbackxml.html
 [Hiera]: /hiera/latest/configuring.html
 [gems]: /puppetserver/latest/gems.html
-[core dependencies]: /puppet/latest/reference/about_agent.html#what-are-puppet-agent-and-puppet-server
+[core dependencies]: /puppet/latest/reference/about_agent.html#what-are-puppet-agent-and-puppetserver
 [environment]: /puppet/latest/reference/environments.html
 [environment caching]: /puppet/latest/reference/configuration.html#environmenttimeout
 
 Starting in version 2.3.0, you can restart Puppet Server by sending a hangup signal, also known as a [HUP signal or SIGHUP](https://en.wikipedia.org/wiki/SIGHUP), to the running Puppet Server process. The HUP signal stops Puppet Server and reloads it gracefully, without terminating the JVM process. This is generally *much* faster than completely stopping and restarting the process. This allows you to quickly load changes to your Puppet Server master, including configuration changes.
 
 There are several ways to send a HUP signal to the Puppet Server process, but the most straightforward is to run the following [`kill`](http://linux.die.net/man/1/kill) command:
 
-    kill -HUP `pgrep -f puppet-server`
+    kill -HUP `pgrep -f puppetserver`
```

Comment:
Yeah, assuming we stick with the new uberjar name of `puppetserver-release.jar` instead of `puppet-server-release.jar`, it would be changing.  With the ezbake package I built from the latest work on the branch, http://builds.puppetlabs.lan/puppetserver/2.4.0.master.SNAPSHOT.2016.04.20T0815/, I did have to do a `pgrep -f puppetserver` instead of `pgrep -f puppet-server` in order to find the id of the running java process for Puppet Server.